src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageSettings/PageSettings.jsx

```diff
@@ -94,8 +94,8 @@ class PageSettings extends Component {
             this.props.onChangeField("hierarchy", parentPageName);
         }
     }
-
-    render() {
+    
+    render_defaultTabsDetails(){
```

Comment:
Where is this used?

Reply:
Just here. I notice this is just a redirect. Changing the implementation of render_defaultTabsDetails() inside render.